tests/functional/a/access/access_attr_before_def_false_positive.py

```diff
@@ -6,35 +6,27 @@
 self.cookedq."""
 
 
-import telnetlib
```

Comment:
I don't think we should modify the functional tests, could you revert this file please ?

Reply:
I expected this revert to fail pre-commit; your suggestion to exclude fixtures must have done the trick.